chia/server/ws_connection.py

```diff
@@ -74,8 +77,11 @@ class WSChiaConnection:
     is_outbound: bool
 
     # Messaging
-    incoming_queue: asyncio.Queue[Tuple[Message, WSChiaConnection]]
+    received_message_callback: Optional[ConnectionCallback]
+    incoming_queue: asyncio.Queue[Message] = field(default_factory=asyncio.Queue)
     outgoing_queue: asyncio.Queue[Message] = field(default_factory=asyncio.Queue)
+    api_tasks: Dict[bytes32, asyncio.Task[None]] = field(default_factory=dict)
```

Comment:
I think some comments here may be warranted. what's the distinction between an "api_task" and an "execute_task"?

Reply:
Resolving, as this PR has since been approved, and comments have been added.